README.md

```diff
@@ -91,5 +91,6 @@ Please check [CONTRIBUTING.md](CONTRIBUTING.md) and the [Developer Guide](docs/d
 
 ## Community
 
-* Join our [Kubeflow Slack Channel](https://kubeflow.slack.com/archives/C06627U3XU3)
+* Join [CNCF Slack Channel](https://www.kubeflow.org/docs/about/community/#kubeflow-slack-channels)
```

Comment:
I think it's better provide the direct link to CNCF Kubeflow Spark Operator channel 
replace the link with this one https://cloud-native.slack.com/archives/C074588U7EG

Reply:
Previously, we got a feedback that if we provide this link and user is not member of CNCF slack, they can't join it: https://github.com/kubeflow/katib/issues/2343

Reply:
Ah, just checked the comment from Andrey on this. Maybe we can rephrase it as:

Join the [CNCF Slack Channel](https://www.kubeflow.org/docs/about/community/#kubeflow-slack-channels) #kubeflow-spark-operator for collaboration on the Spark Operator.

Reply:
that sounds good

Reply:
@vara-bonthu @andreyvelich please take a look again, I have updated the same. Happy to put a link to kubeflow-spark-operator as well if you think it would make more sense. I am double minded about it as it ma confuse users and they may not end up clicking on first link to join the CNCF Slack Workspace